morus:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 10

  # Angle of Tilt Controllers ---------------------------------
  angle_tilt_0_controller:
    type: effort_controllers/JointPositionController
    joint: tilt_0_joint
    pid: {p: 100., i: 15., d: 50., i_clamp_max: 1.0, i_clamp_min: -1.0}
  angle_tilt_1_controller:
    type: effort_controllers/JointPositionController
    joint: tilt_1_joint
    pid: {p: 100., i: 15., d: 50., i_clamp_max: 1.0, i_clamp_min: -1.0}
  angle_tilt_2_controller:
    type: effort_controllers/JointPositionController
    joint: tilt_2_joint
    pid: {p: 100., i: 15., d: 50., i_clamp_max: 1.0, i_clamp_min: -1.0}
  angle_tilt_3_controller:
    type: effort_controllers/JointPositionController
    joint: tilt_3_joint
    pid: {p: 100., i: 15., d: 50., i_clamp_max: 1.0, i_clamp_min: -1.0}
